pallas-common/src/test/java/com/vip/pallas/utils/IPUtilsTest.java

```diff
@@ -0,0 +1,69 @@
+package com.vip.pallas.utils;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import java.net.InetSocketAddress;
+import java.net.UnknownHostException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+
+public class IPUtilsTest {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void testGetHostAddress() {
+    assertNull(IPUtils.getHostAddress(null));
+    assertEquals("192.168.1.1:255", IPUtils.getHostAddress(new InetSocketAddress("192.168.1.1", 255)));
+  }
+
+  @Test
+  public void testGetHostName() {
+    assertEquals("127.0.1.1", IPUtils.getHostName("127.0.0.1"));
+    assertEquals("192.168.1.1", IPUtils.getHostName("192.168.1.1:255"));
+  }
+
+  @Test
+  public void testGetHostNameAndPort() {
+    assertEquals("192.168.1.1:255", IPUtils.getHostNameAndPort("192.168.1.1:255"));
+  }
+
+  @Test
+  public void testGetHostAndPort() {
+    assertEquals("192.168.1.1:-1", IPUtils.getHostAndPort("192.168.1.1").toString());
+    assertEquals("192.168.1.1:255", IPUtils.getHostAndPort("192.168.1.1:255").toString());
+    assertEquals("24:-1", IPUtils.getHostAndPort("192.168.1.1/24").toString());
+    assertEquals("null:-1", IPUtils.getHostAndPort("").toString());
+  }
+
+  @Test
+  public void testGetHostAndPort2() {
+    assertEquals("192.168.1.1:255", IPUtils.getHostAndPort(
+            new InetSocketAddress("192.168.1.1", 255)).toString());
+    final InetSocketAddress inetSocketAddress = null;
```

Comment:
(line 46) could `null` be inlined in the call to getHostAndPort()?

Reply:
It could be but that line is there to test the case when the InetSocketAddress is null as this does add more code coverage